src/test/java/ControllerTest/GameControllerTest.java

```diff
@@ -1,19 +1,23 @@
 package ControllerTest;
 
 import Controller.GameController;
+import Model.Board;
 import Model.Field;
+import Model.PlayerTurn;
 import javafx.scene.paint.Color;
 
 import static javafx.scene.paint.Color.rgb;
 
 public class GameControllerTest {
+    protected Board board;
     protected GameController controller;
     protected Field[][] fields;
     protected final Color white = rgb(255, 255, 255);
     protected final Color black = rgb(0, 0, 0);
 
-    protected void initialize(GameController controller) {
+    protected void initialize(Board board,GameController controller) {
+        this.board = board;
         this.controller = controller;
-        this.fields = controller.getBoard().getFields();
+        this.fields = board.getFields();
     }
 }
```

Comment:
Czy można to uprościć i zostawić tylko jeden parametr GameController controller, a this.board ustawić jako controller.getBoard()?